vdr/cmd/cmd.go

```diff
@@ -153,6 +155,24 @@ func resolveCmd() *cobra.Command {
 	return result
 }
 
+func deactivateCmd() *cobra.Command {
+	result := &cobra.Command{
+		Use:   "deactivate [DID]",
+		Short: "Deactivate a DID document based on its DID",
```

Comment:
maybe mention this is a destructive operation and it can't be reverted

Reply:
Added a confirmation 👍